test/schema/artist/index.js

```diff
@@ -264,9 +264,110 @@ describe('Artist type', () => {
       const query = `
         {
           artist(id: "foo-bar") {
+            blurb
+          }
+        }
+      `;
+
+      return runQuery(query)
+        .then(data => {
+          expect(data).to.eql({
+            artist: {
+              blurb: 'catty blurb',
+            },
+          });
+        });
+    });
+  });
+
+  describe('biography_blurb', () => {
+    describe('with partner_bio set to true', () => {
+      describe('with a featured partner bio', () => {
+        beforeEach(() => {
+          Artist.__ResetDependency__('gravity');
+          const gravity = sinon.stub();
+          Artist.__Rewire__('gravity', gravity);
+          gravity
+            // Artist
+            .onCall(0)
+            .returns(Promise.resolve(assign({}, artist)))
+            // PartnerArtist
+            .onCall(1)
+            .returns(Promise.resolve([assign({}, {
+              biography: 'new catty bio',
+              partner: { name: 'Catty Partner', id: 'catty-partner' },
+            })]));
+        });
+        afterEach(() => {
+          const query = `
+            {
+              artist(id: "foo-bar") {
+                biography_blurb(partner_bio: true) {
+                  text
+                  credit
+                  partner_id
+                }
+              }
+            }
+          `;
+
+          return runQuery(query)
+            .then(data => {
+              expect(data).to.eql({
+                artist: {
+                  biography_blurb: {
+                    text: 'new catty bio',
+                    credit: 'Submitted by Catty Partner',
+                    partner_id: 'catty-partner',
+                  },
+                },
+              });
+            });
+        });
+        it('returns the featured partner bio without an artsy blurb', () => {
+        });
+
+        it('returns the featured partner bio with an artsy blurb', () => {
+          artist.blurb = 'artsy blurb';
+        });
```

Comment:
Oh, I see, you’re running the actual tests in `afterEach` and using the specs to setup the data? Very interesting, not really what I meant earlier, tho, but we can go over that after this is merged, as long as the changes are being tested 👌